report/sections/sprint3/usability-test-sprint-3.tex

```diff
@@ -0,0 +1,76 @@
+After this hurdle, he performed the task without issues.
+\\\\
+For assignment 8, he simply did not notice the log out button initially, causing it to take a little longer.
+After completing the assignments we asked Emil to fill out two regular days, such that we could examine how long it would take and what a typical day would look like. 
+Filling out a day with 18-20 activities took Emil around three minutes.
+
+\subsection{Clarifying questions}
+Upon completing the test we posed some clarifying questions to examine his opinion of the program.
+The overall design was intuitive according to Emil. He thought it was relatively simple, and not too busy so as to be overwhelming for the citizens.
+He felt the assignments were easily completed because of the intuitiveness of the program, and for the assignments that caused trouble he mainly blamed the issues on it being the first time he used the program, rather than it being poorly implemented.
+In terms of improvements he would like to see changes to the drag and drop functionality of moving activities around.
+He felt it was hard to determine where the activity would land if it were to be dragged between two other activities.
+A visual representation of where it would end up, such as a line indicating it, would therefore be preferable.
+\\\\
+Because we were uncertain, we asked Emil how many citizens he or his colleagues would generally create week plans for.
+He responded by saying that, at the moment, he only has one citizen, but the absolute maximum at Egebakken would be four.
+Choice boards were considered for implementation by the previous year on the GIRAF project.
+These boards should allow for multiple activities to be added, and then the citizen would select one of them.
+Emil thinks this would be important to teach the citizens that they also have an influence, rather than just having them do what they are told all the time.
+As such, this could be a feature to implement in the future as it was still deemed attractive.
+\\\\
+Emil was initially a proponent for the implementation of a timer functionality for the GIRAF project, as seen in \autoref{interview-with-emil}.
+We were uncertain whether the citizen or the guardian would be the one to change representation shown for the citizen.
+Emil thought that this should generally be done by guardians.
+The citizens can have issues in telling time intervals apart, and choosing the proper representation would require some knowledge that only the guardian would have.
+Once started, whether or not a citizen should be able to pause a timer was posed as a question.
+He thought that both options had their merits. 
+As such, he would like for a type of lock to be implemented on the timer.
+The guardian would be able to choose that the timer should be locked when adding it, making it unable to be paused.
+
+\subsection{Prototype feedback}
```

Comment:
Der må meget gerne skrives omkring, at han havde valgmuligheden mellem to prototyper ved kopiering af ugeplan, og hvilken en han valgte.

Lav en autoref til figuren i Andreas' afsnit hvor vi sammenligner de to prototyper